servers.conf.erb

```diff
@@ -22,6 +22,15 @@ server {
     proxy_set_header X-Real-IP $remote_addr ;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for ;
   }
+  #
+  # Reroute versions to backend
+  location ~ ^/versions/[^/]+/backend/(.*)$ {
+    rewrite ^/versions/[^/]+/backend/(.*)$ /$1 break;
+    proxy_pass http://localhost:8002;
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+  }
```

Comment:
I have a hunch we should come with a more general approach with double slash handling… btw, why isn't the nginx `merge_slashes` directive, why is enabled by default, work in our case? 🤔

Reply:
Here the problem I’m solving is not the double slash ending. It’s the fact that http calls to `/versions/*/backend/*` should be sent to the Litestar backend (localhost:8002) and the path rewritten to keep only the part after the `/backend/` string (Litestar doesn’t know anything before the `/api`).